README.md

```diff
@@ -64,19 +64,57 @@ ckan db upgrade
 ```
 to execute the database migration script
 
-Also there is CLI command to initialize the table:
+## CLI
+
+
+There is CLI command to initialize the table:
 ```
 ckan -c /path/to/ckan.ini datastore_config init_db
 ```
-Configuration is done by the CKAN admin menu
-
+You can execute refresh datastore using the command `refresh_dataset_datastore` and passing the frequency interval in minutes as parameter. The frequency value must be defined in the `frequency_options.json` configuration file. This will reupload the data from resource to datastore
 ```
-@hourly datastore_config -c /path/to/ckan.ini refresh_dataset_datastore 10
+@hourly ckan -c /path/to/ckan.ini datastore_config refresh_dataset_datastore 10
 ```
 10 - frequency  to refresh the datastore  (in minutes)
-Cron jobs can be set by the desired frequency which currently is set to 10 min, 2 hours or 24 hours
+Cron jobs can be set by the desired frequency which currently is set to 10 min, 2 hours(120 minutes) or 24 hours(1440 minutes).
```

Comment:
Cron jobs can be set by the desired frequency, you can find all available frequencies  at ``frequency_options.json`` or you can list them via ``ckan -c /path/to/ckan.ini datastore_config available_choices``